src/js/components/Table/Paginated/DateFilter.tsx

```diff
@@ -65,6 +68,13 @@ const formatDate = (x?: string) => {
   return date;
 };
 
+const formatDisplayDate = (date: string): string => {
+  if (navigator.language) {
+    return new Intl.DateTimeFormat(navigator.language).format(formatDate(date));
```

Comment:
Do we want to allow people to pass in another language code instead of just relying on what's in the navigator?

We also probably want to use navigator.languages if we're having opinions about the date formatting in TeamSnap UI. There's always the option to pass in the language code to TeamSnap UI components and let the consumers decide which language to use :)

Here's a good post explaining how language choosing is difficult: https://phrase.com/blog/posts/detecting-a-users-locale/

That post mentions using `navigator.languages` over `navigator.language` and I believe the DateTimeFormat accepts an array of language codes, so we may just want to use that. Unfortunately, https://caniuse.com/?search=navigator.languages shows that we'll lose a bit of IE support if we take this approach, but I think that's okay given our supported browsers (side bar, we should totally document our supported browsers somewhere)

Reply:
Made a few updates to this and would like your input on the changes.